receiver/kafkareceiver/testdata/config.yaml

```diff
@@ -4,17 +4,8 @@ kafka:
     - "foo:123"
     - "bar:456"
   resolve_canonical_bootstrap_servers_only: true
-  client_id: otel-collector
```

Comment:
could we keep this to make sure the API does not change?

Reply:
Do you mean auth and metadata? I removed them because we have coverage for the ClientConfig type in the internal/kafka/configkafka package. We're still verifying client_id and group_id below, as a minimal subset of the CientConfig and ConsumerConfig types - to verify that we're embedding them correctly.

Reply:
Yeah, it would be great to know which API you mean here @pavolloffay